.changeset/poor-bobcats-sort.md

```diff
@@ -0,0 +1,55 @@
+---
+'@ai-sdk/ai': major
+'@ai-sdk/amazon-bedrock': major
+'@ai-sdk/azure': major
+'@ai-sdk/deepinfra': major
+'@ai-sdk/fal': major
+'@ai-sdk/fireworks': major
+'@ai-sdk/google-vertex': major
+'@ai-sdk/langchain': major
+'@ai-sdk/llamaindex': major
+'@ai-sdk/luma': major
+'@ai-sdk/openai-compatible': major
+'@ai-sdk/openai': major
+'@ai-sdk/react': major
+'@ai-sdk/replicate': major
+'@ai-sdk/rsc': major
+'@ai-sdk/svelte': major
+'@ai-sdk/togetherai': major
+'@ai-sdk/vue': major
+'@ai-sdk/xai': major
+'ai': major
```

Comment:
fyi this broke the build and included unaffected/non-existing packages. and was missing the provider pkg

Reply:
https://github.com/vercel/ai/pull/6231

Reply:
sorry for that, I'll be more careful. We could add a CI check for this. There doesn't seem to be a `--dry-run` option, but someone suggested a workaround: https://github.com/changesets/changesets/issues/614#issuecomment-2828712913